CHANGELOG.md

```diff
@@ -10,6 +10,8 @@ available [on GitHub][2].
 
 <!-- Add new changes here -->
 
+`done` command has been added by [@sloboegen].
```

Comment:
Thanks a lot for updating the changelog! 🙏🏻 

Can you also add your GitHub user name to the `<!-- Contributors -->` section below?

I'm afraid it won't work as a link as is 😞